packages/app-content-pages/src/screens/About/components/Stats/Stats.js

```diff
@@ -0,0 +1,89 @@
+import { Box, ResponsiveContext, Text } from 'grommet'
+import { useContext } from 'react'
+import { useTranslation } from 'next-i18next'
+import styled from 'styled-components'
+import SpacedText from '@zooniverse/react-components/SpacedText'
+import AnimtedNumber from '@zooniverse/react-components/AnimatedNumber'
+
+import { useTotalClassificationCount, useTotalVolunteerCount } from './hooks'
+
+const ClassificationsLabel = styled(Box)`
+  background: linear-gradient(
+    to right,
+    rgba(240, 178, 0, 0.4) 0%,
+    rgba(240, 178, 0, 1) 25%,
+    rgba(240, 178, 0, 1) 75%,
+    rgba(240, 178, 0, 0.4) 100%
+  );
+`
+
+const VolunteersLabel = styled(Box)`
+  background: linear-gradient(
+    to right,
+    rgba(0, 93, 105, 0.4) 0%,
+    rgba(0, 93, 105, 1) 25%,
+    rgba(0, 93, 105, 1) 75%,
+    rgba(0, 93, 105, 0.4) 100%
+  );
+`
+
+const Stat = styled(Box)`
+  overflow: hidden;
+
+  &.classifications {
+    background: rgba(240, 178, 0, 0.1); // matches ClassfiicationsLabel
+  }
+
+  &.volunteers {
+    background: rgba(0, 93, 105, 0.1); // matches VolunteersLabel
+  }
+`
+
+/*
+  We have to manually add some legacy stats to the UI.
+  These numbers are not included in the fetch response for
+  total volunteers or classifications.
+  https://github.com/zooniverse/Panoptes-Front-End/pull/4511
+*/
+const GZ123_COUNT = 98989226 // classifications
+const OUROBOROS_COUNT = 142800311 // classifications
+const OTHERS_COUNT = 8680290 // classifications
+const OUROBOROS_USER_COUNT = 124921 // volunteers
```

Comment:
Are these constants because they're no longer getting active incremental value? Or are these going to be updated at some interval? Hard to tell from comment.

Reply:
I added the word "static" to the inline comment above them. More info can be found in the linked PFE PR: https://github.com/zooniverse/Panoptes-Front-End/pull/4511